README.md

```diff
@@ -31,6 +32,7 @@ Table of Contents <!-- omit in toc -->
   - [MacOS GUI applications in Docker](#macos-gui-applications-in-docker)
 - [Citation](#citation)
 - [License](#license)
+- [Documentation](#documentation)
```

Comment:
Please also move this up in the table of contents

Reply:
Done 👍🏻